share/instruments/instruments.xml

```diff
@@ -7056,7 +7056,7 @@
                         <head>normal</head>
                         <line>0</line>
                         <voice>0</voice>
-                        <name>Acoustic Snare</name>
+                        <name>Snare</name>
```

Comment:
Might need adjustments in ../importexport/guitarpro/importgtp.cpp and ../libmscore/drumset.cpp:253

Reply:
If not, we'd need to enhance the generateTS.py ro also extract those strings from instruments.xml, else it's end up untranslated